src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -497,12 +493,8 @@ def fn(
             return fn
 
         get_event_stats = get_event_stats_factory(dataset)
-        zerofill_results = not (
-            request.GET.get("withoutZerofill") == "1" and has_chart_interpolation
-        )
-        if use_rpc:
-            # The rpc will usually zerofill for us so we don't need to do it ourselves
-            zerofill_results = False
```

Comment:
ow my head. 
Naming things like `NotFoo` or `WithoutBar` makes logic puzzles like this hard.

But it boils down to a few steps:
| Step | Code |
| --- | --- |
| replace the var with False, because it's off all the time |  `zerofill_results = not (request.GET.get("withoutZerofill") == "1" and False)`
| simplify the braces | `zerofill_results = not (False)
| simplify the `not` | `zerofill_results = True`
| combine with `use_rpc` | `zerofill_results = not use_rpc`